src/pdm/core.py

```diff
@@ -78,12 +78,11 @@ def init_parser(self) -> None:
             "--config",
             help="Specify another config file path [env var: PDM_CONFIG_FILE] ",
         )
-        self.parser._positionals.title = "Commands"
         verbose_option.add_to_parser(self.parser)
         ignore_python_option.add_to_parser(self.parser)
         pep582_option.add_to_parser(self.parser)
 
-        self.subparsers = self.parser.add_subparsers(parser_class=ArgumentParser, dest="fooo")
+        self.subparsers = self.parser.add_subparsers(parser_class=ArgumentParser, title="commands", metavar="")
```

Comment:
commands is the default title?

Reply:
Yes.

In previous revision, the 81L `self.parser._positionals.title` was set to `Commands`, force argparse displays "Commands:" as title instead of "Positional Arguments:". But it can be done by setting subparsers title to "commands", as same as other venv/self/cache subparsers, without touching the internal members.